src/libs/ReportUtils.ts

```diff
@@ -1364,49 +1442,42 @@ function getReimbursementQueuedActionMessage(reportAction, report) {
 /**
  * Returns the last visible message for a given report after considering the given optimistic actions
  *
- * @param {String} reportID - the report for which last visible message has to be fetched
- * @param {Object} [actionsToMerge] - the optimistic merge actions that needs to be considered while fetching last visible message
- * @return {Object}
+ * @param reportID - the report for which last visible message has to be fetched
+ * @param [actionsToMerge] - the optimistic merge actions that needs to be considered while fetching last visible message
+
  */
-function getLastVisibleMessage(reportID, actionsToMerge = {}) {
+function getLastVisibleMessage(reportID: string | undefined, actionsToMerge: ReportActions = {}): LastVisibleMessage {
     const report = getReport(reportID);
-    const lastVisibleAction = ReportActionsUtils.getLastVisibleAction(reportID, actionsToMerge);
+    const lastVisibleAction = ReportActionsUtils.getLastVisibleAction(reportID ?? '', actionsToMerge);
 
     // For Chat Report with deleted parent actions, let us fetch the correct message
-    if (ReportActionsUtils.isDeletedParentAction(lastVisibleAction) && isChatReport(report)) {
+    if (ReportActionsUtils.isDeletedParentAction(lastVisibleAction) && report && isNotEmptyObject(report) && isChatReport(report)) {
```

Comment:
```suggestion
    if (ReportActionsUtils.isDeletedParentAction(lastVisibleAction) && isNotEmptyObject(report) && isChatReport(report)) {
```
This is not required.